.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
+
+---
+<!-- Issues are for bugs and feature requests** only -->
+
+##### ISSUE TYPE
+
+- Bug Report
+
+##### SUMMARY
+<!-- Briefly describe the problem. -->
+
+##### STEPS TO REPRODUCE
+
+<!-- Please describe exactly how to reproduce the problem. -->
+
+##### EXPECTED RESULTS
+
+<!-- What did you expect to happen when running the steps above? -->
+
+##### ACTUAL RESULTS
+
+<!-- What actually happened? -->
+
+##### ADDITIONAL INFORMATION
+
+<!-- Include any links to sosreport, database dumps, screenshots or other
```

Comment:
`sosreport`? We use RedHat stuff in this project? :stuck_out_tongue_closed_eyes:

Reply:
I change the comment